### DISCUSSION TITLE
Compose regression

### DISCUSSION THREAD

**Discussion by robacarp** [AllYourBot/hostedgpt#442] [General]
I'm getting the following error message from `docker compose up`

```
validating docker-compose.yml: services.base.environment array items[4,10] must be unique
```

Which may be a kind of merge conflict introduced in 5e948f3842ac8ce6e19308cd153f2f63f34386c0 by @matthewbennink or in 43e9eb2648e4a3fea973e315b1a5401f4ced8153 by @krschacht

**Comment by keithschacht** [comment]
@robacarp Ugh, I keep breaking one of the many docker configs. I just committed a hotfix to remove that duplicate key.

Usually I break a docker config for one of our hosting environments, since those aren’t auto tested, but we have docker build CI tests so I wonder why it didn’t catch this one? In any case, should be fixed now

**Reply by robacarp** [reply]
I believe all of the docker build tests are straight docker, not docker-compose, right?

**Reply by keithschacht** [reply]
Oh, yes, that would explain it.
